src/libraries/System.Runtime/tests/System.Runtime.Tests/System/Int32Tests.cs

```diff
@@ -663,6 +663,46 @@ public static IEnumerable<object[]> Parse_Invalid_TestData()
             yield return new object[] { "2147483649-", NumberStyles.AllowTrailingSign, null, typeof(OverflowException) };
             yield return new object[] { "(2147483649)", NumberStyles.AllowParentheses, null, typeof(OverflowException) };
             yield return new object[] { "2E10", NumberStyles.AllowExponent, null, typeof(OverflowException) };
+
+            // Test trailing non zeros
+
+            yield return new object[] { "-9223372036854775808.1", NumberStyles.Number, CultureInfo.InvariantCulture, typeof(OverflowException) };
```

Comment:
Just noting for reviewers that this `Parse_Invalid_TestData` is reused by the other integer types as part of their parsing validation logic.

For example: https://github.com/dotnet/runtime/blob/b4d433380c24889565cc8ea7703db1106c2ad0df/src/libraries/System.Runtime/tests/System.Runtime.Tests/System/SByteTests.cs#L266-L269

and 

https://github.com/dotnet/runtime/blob/b4d433380c24889565cc8ea7703db1106c2ad0df/src/libraries/System.Runtime/tests/System.Runtime.Tests/System/Int64Tests.cs#L303-L313